frontend/src/utils/dataset.ts

```diff
@@ -205,9 +204,7 @@ export async function searchDatasets<T = Dataset>({
   endpoint += `&include_archived=${!!options.showArchived}`;
   endpoint += `&include_drafts=${!!options.includeDrafts}`;
   console.log(endpoint)
-  const response = await CkanRequest.get<any>(endpoint, {
-    headers: { Authorization: apiKey },
-  });
+  const response = await CkanRequest.get<any>(endpoint);
```

Comment:
You can't do it, you need to send the apiKey when it's received because we need to return datasets that are private for users that belong to the organizations.

Reply:
fixed